package.json

```diff
@@ -1,7 +1,7 @@
 {
   "devDependencies": {
     "@types/jest": "25.2.3",
-    "@types/node": "12.12.6",
+    "@types/node": "14.14.36",
```

Comment:
- When we originally started this project node.js 12 was the newest LTS version and still has more than a year of support remaining
- Questions:
  - Is their a compelling reason to make the jump to v14 from v12 now?
  - As it appears you completed the upgrade to v14 on the SEA, any major or breaking changes or adjustments required?